helpers/deploymentConfig.ts

```diff
@@ -2545,10 +2549,10 @@ export const globalConfig: NetworkConfig = {
             asset: "USDT",
             symbol: "vUSDT_Tron",
             rateModel: InterestRateModels.JumpRate.toString(),
-            baseRatePerYear: convertToUnit("0.03", 18),
-            multiplierPerYear: convertToUnit("0.1", 18),
+            baseRatePerYear: convertToUnit("0.02", 18),
+            multiplierPerYear: convertToUnit("0.2", 18),
             jumpMultiplierPerYear: convertToUnit("2.5", 18),
-            kink_: convertToUnit("0.6", 18),
```

Comment:
Were changes to baseRatePerYear and kink_ recommended?

Reply:
Understood, updated based on the actual configuration.

Reply:
I think the model used currently has the new values, it just wasn't updated here https://bscscan.com/address/0x009cdFB248e021f58A34B50dc2A7601EA72d14Ac#readContract